src/DataStructures/Python/Matrix.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/python.hpp>
+#include <sstream>
+#include <string>
+#include <utility>
+
+#include "DataStructures/Matrix.hpp"
+#include "PythonBindings/BoundChecks.hpp"
+#include <boost/python/reference_existing_object.hpp>
+#include <boost/python/return_value_policy.hpp>
+#include <boost/python/tuple.hpp>
+namespace bp = boost::python;
+
+namespace py_bindings {
+void bind_matrix() {
+  // Wrapper for basic Matrix operations
+  bp::class_<Matrix>("Matrix", bp::init<size_t, size_t>())
+    .def("size",
+         +[](const Matrix& M){
+           bp::tuple a= bp::make_tuple<size_t,size_t>(M.rows(), M.columns());
```

Comment:
just do `return bp::make_tuple...` (also your formatting needs work :) ) I think you can run `git clang-format` and `clang-format` on Wheeler